tests/test_type.py

```diff
@@ -7,15 +7,28 @@
 from tests import db
 
 
+# Since we set all table_name to cte_hex(), we can't use regclass(t["tableoid"]) anymore
```

Comment:
Does Greenplum have lots of operators or functions which will access to `tableoid`?

Reply:
I will just remove this comments